_layouts/downloads.html

```diff
@@ -30,24 +30,30 @@ <h1 class="title-bg">{{site.data.pages.es.downloads.elasticsearch.title}}</h1>
         <!-- First row of images -->
         <div class="row justify-content-center text-center downloadsContainer">
 
-          <div class="col-6 col-md-4"><img src="{{site.baseurl}}{{site.data.pages.es.downloads.images.rpm}}" class="download-img"> </div>
+          <div class="col-6 col-md-4"><img src="{{site.baseurl}}{{site.data.pages.es.downloads.images.rpm}}"
+                                           class="download-img"></div>
 
-          <div class="col-6 col-md-4"><img src="{{site.baseurl}}{{site.data.pages.es.downloads.images.docker}}" class="download-img"> </div>
+          <div class="col-6 col-md-4"><img
+                  src="{{site.baseurl}}{{site.data.pages.es.downloads.images.docker}}" class="download-img">
+          </div>
 
         </div>
 
         <!-- First row of downloads -->
         <div class="row justify-content-center text-center px-2">
-          <div class="col-12 col-md-10"> </div>
+          <div class="col-12 col-md-10"></div>
           <div class="col-6 col-md-4 pt-3 pb-5 link-on-white-paragraph">
-            <a href="{{site.data.pages.es.downloads.elasticsearch.rpm_url}}" class="software-download" target="about:blank" onclick="trackOutboundLink('{{site.data.pages.es.downloads.elasticsearch.rpm_url}}');">
-              <h7 style="font-weight:bold;">RPM install guide</h7>
+            <a href="{{site.data.pages.es.downloads.elasticsearch.rpm_url}}" class="software-download"
+               target="about:blank"
+               onclick="trackOutboundLink('{{site.data.pages.es.downloads.elasticsearch.rpm_url}}');">
+              <h7 style="font-weight:bold;">RPM Install guide</h7>
```

Comment:
"Install guide" is capitalized differently throughout this page. We use sentence case whenever possible, so RPM install guide, Debian install guide, Install guide, etc.

Reply:
I have made the suggested changes.